src/web/components/Callout/FileUpload.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { FieldLabel } from './FieldLabel';
+
+const fileUploadInputStyles = css`
+    padding-top: 10px;
+    padding-bottom: 10px;
+`;
+
+type Props = {
+    formField: CampaignFieldFile;
+    formData: { [key in string]: any };
```

Comment:
Is it possible to type this data better? I think we can predict what the data will be so it'd be great to build the type for it

Reply:
`formData` is constructed on data that is fetched at runtime, there is no way to know what those keys will be beforehand as a result

Reply:
How about if we check the code that builds the form data to see what will be sent back and use that to construct a contract? Or, we just work out the contract that we need to prevent breaking the form and demand that this be met.

If we don't set any contract at all then we have the worst of both worlds. We are open to errors and have no contract to learn from. At least if we set a contract there could still be errors (if the backend doesn't meet it) but we're on the way to being properly typed.

Reply:
The only reason we pass down `formData` is so we can update the parent state object. Technically there is minimal relation between `formData` and `FileUpload`.
We could remove it as a prop an instead pass down a function that would only update the `formField.id` relevant in `formData` ?